addon/services/scheduler.js

```diff
@@ -164,3 +165,34 @@ export default Service.extend({
     run.cancel(this._nextPaintTimeout);
   }
 });
+
+if (DEBUG) {
+  Scheduler.reopen({
+    init() {
+      this._super(...arguments);
+
+      if (Ember.testing) {
+        this._waiter = () => {
+          if (!this.queues) {
+            return;
+          }
+          const lastQueueName = this.queueNames[this.queueNames.length - 1];
```

Comment:
Any reason to not just use `'afterContentPaint'` here? I know this is dynamic, but our queue names are actually fixed.

Reply:
no particular reason - I just wanted to not have to update the waiter code in case someone added another queue after afterContentPaint. What would you recommend here?

Reply:
Let's just leave this for now as this conforms to the current API. I think we can likely do some refactoring of this lib to clean some things up. I'll look at doing that in a subsequent PR.

Thanks!